src/test/ui/consts/const-eval/simd/insert_extract.rs

```diff
@@ -2,7 +2,7 @@
 #![feature(repr_simd)]
 #![feature(platform_intrinsics)]
 #![feature(staged_api)]
-#![stable(feature = "foo", since = "1.33.7")]
+#![stable(feature = "foo", since = "1.3.37")]
```

Comment:
Any particular reason for this change?

Reply:
```console
error[E0711]: feature `foo` is declared stable since 1.3.37, but was previously declared stable since 1.33.7
  --> /git/rust/src/test/ui/consts/const-eval/simd/insert_extract.rs:15:5
   |
LL |     #[rustc_const_stable(feature = "foo", since = "1.3.37")]
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0711]: feature `foo` is declared stable since 1.3.37, but was previously declared stable since 1.33.7
  --> /git/rust/src/test/ui/consts/const-eval/simd/insert_extract.rs:17:5
   |
LL |     #[rustc_const_stable(feature = "foo", since = "1.3.37")]
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors
```

Reply:
Ah, didn't look at the full context.